packages/blocks/src/page-block/edgeless/edgeless-page-block.ts

```diff
@@ -1115,10 +1118,26 @@ export class EdgelessPageBlockComponent
     );
   }
 
+  private _initFontloader() {
+    if (!this.fontLoader) this.fontLoader = new FontLoader();
+
+    this._disposables.add(
+      this.fontLoader.slots.loaded.on(font => {
+        if (font !== 'Kalam:n4,n7' || !this.surface) return;
```

Comment:
What's `Kalam:n4,n7`?

Reply:
https://fonts.google.com/specimen/Kalam

Reply:
"n" means `font-style: normal`, "4" and "7" means `font-weight: 400` and `font-weight: 700`